[release/1.6] `error: implicit declaration of function ‘memcpy’`

### Description

v1.6 cannot be built due to: `error: implicit declaration of function ‘memcpy’`

### Steps to reproduce the issue

```
git checkout v1.6.38
make
```

### Describe the results you received and expected

Received:
```console
$ make
+ bin/ctr
+ bin/containerd
# github.com/containerd/btrfs
btrfs.c: In function ‘unpack_root_item’:
btrfs.c:25:9: error: implicit declaration of function ‘memcpy’ [-Wimplicit-function-declaration]
   25 |         memcpy(dst->uuid, src->uuid, BTRFS_UUID_SIZE);
      |         ^~~~~~
btrfs.c:23:1: note: include ‘<string.h>’ or provide a declaration of ‘memcpy’
   22 | #include "btrfs.h"
  +++ |+#include <string.h>
   23 | 
btrfs.c:25:9: warning: incompatible implicit declaration of built-in function ‘memcpy’ [-Wbuiltin-declaration-mismatch]
   25 |         memcpy(dst->uuid, src->uuid, BTRFS_UUID_SIZE);
      |         ^~~~~~
btrfs.c:25:9: note: include ‘<string.h>’ or provide a declaration of ‘memcpy’
make: *** [Makefile:254: bin/containerd] Error 1
```

Expected: no error

### What version of containerd are you using?

v1.6.38

### Any other relevant information

Ubuntu 25.04
```console
$ dpkg -l gcc
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version           Architecture Description
+++-==============-=================-============-=================================
ii  gcc            4:14.2.0-1ubuntu1 arm64        GNU C compiler
```

### Show configuration if it is related to CRI plugin.

_No response_

--- a/go.mod
+++ b/go.mod
@@ -8,7 +8,7 @@ require (
 	github.com/Microsoft/go-winio v0.5.3
 	github.com/Microsoft/hcsshim v0.9.12
 	github.com/containerd/aufs v1.0.0
-	github.com/containerd/btrfs v1.0.0
+	github.com/containerd/btrfs v1.0.1
 	github.com/containerd/cgroups v1.0.4
 	github.com/containerd/console v1.0.3
 	github.com/containerd/continuity v0.3.0

